src/main/java/org/springframework/data/couchbase/core/convert/OtherConverters.java

```diff
@@ -90,4 +93,24 @@ public BigInteger convert(String source) {
 			return source == null ? null : new BigInteger(source);
 		}
 	}
+
+	@WritingConverter
+	public enum BigDecimalToString implements Converter<BigDecimal, String> {
+		INSTANCE;
+
+		@Override
+		public String convert(BigDecimal source) {
+			return source == null ? null : source.toPlainString();
```

Comment:
Is that how other libraries also do  it instead of using toString()? So serializing it without an exponent field

Reply:
That's a really good catch.  That would be incompatible with non- spring data written BigDecimal.